app/scripts/utils/interval-tree.js

```diff
@@ -118,8 +118,8 @@ IntervalTree.prototype.contains = function _contains(point) {
 };
 
 function intersects(a, b) {
-  return (a[0] <= b[0] && a[1] >= b[0]) || (a[0] <= b[1] && a[1] >= b[1])
-      || (b[0] <= a[0] && b[1] >= a[0]) || (b[0] <= a[1] && b[1] >= a[1]);
```

Comment:
What's the functional difference? Are the borders excluded? Just out of curiosity I am wondering, was this a bug or is this just a different choice of behavior.

Reply:
Yeah, borders are excluded. The function difference is minor but very visible. The bedlike track stacks overlapping intervals. So imagine that you have two intervals:

```
|--|
```
and
```
    |--|
```

If we counted borders as intersects then they would be rendered like this:

```
|--|
     |--|
```

We would prefer to render them as adjacent to each other:

```
|--|--|
```

Does that make sense?

Reply:
It makes complete sense! Thanks for the explanation :) It's not necessary but maybe we can save the comment into the function's docstring so we don't forget about the rationale? It's up to you.